pages/_app.js

```diff
@@ -3,6 +3,7 @@ import Layout from "../components/layout/Layout";
 import { GraphQLProvider } from "../context/ApolloContext";
 import { AuthProvider } from "../context/AuthContext";
 import "../styles/globals.css";
+import "../styles/catalogo.css";
```

Comment:
Consider using CSS Modules: https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css. Currently, it is importing those `catalogo.css` styles globally, which on the long run will reduce performance.